src/components/home/elements/TokenRecord.vue

```diff
@@ -0,0 +1,460 @@
+          </template>
+          <b-dropdown-item @click="showModal">Get Token Info</b-dropdown-item>
+          <b-dropdown-item @click="sendToken">Send Token</b-dropdown-item>
+          <b-dropdown-item @click="issueToken">Issue Token</b-dropdown-item>
+          <b-dropdown-item @click="burnToken">Destroy Token</b-dropdown-item>
+          <b-dropdown-item @click="removeToken">Remove Token</b-dropdown-item>
+        </b-dropdown>
+      </b-col>
+    </b-row>
+    <TokenInfoModal :token-id="tokenId"
+                    :issuer="issuer"
+                    :unity="unity"
+                    :maker="maker"
+                    :contract-id="contractId"
+                    :max-supply="maxSupply"
+                    :current-supply="formatter(currentSupply)"
+                    :token-description="tokenDescription">
+    </TokenInfoModal>
+    <IssueToken :token-id="tokenId"
+                :issuer="issuer"
+                :address="address"
+                :wallet-type="walletType"
+                :addresses="addresses"
+                :cold-addresses="coldAddresses"
+                :token-balance="tokenBalance"
+                :balance="balances[address]"
+                :max-supply="maxSupply"
+                :current-supply="currentSupply"
+                :token-unity="unity"
+                @updateBalance="updateBalance">
+    </IssueToken>
+    <SendToken :token-id="tokenId"
+               :token-balances="tokenBalances"
+               :balances="balances"
+               :cold-addresses="coldAddresses"
+               :addresses="addresses"
+               :selected-address="address"
+               :wallet-type="walletType"
+               :function-index="functionIndex"
+               :token-unity="unity"
+               @endSendSignal="endSendSignal">
+    </SendToken>
+    <BurnToken :token-id="tokenId"
+               :issuer="issuer"
+               :address="address"
+               :wallet-type="walletType"
+               :addresses="addresses"
+               :cold-addresses="coldAddresses"
+               :token-balance="tokenBalance"
+               :balance="balances[address]"
+               :token-unity="unity"
+               @updateBalance="updateBalance"></BurnToken>
+  </b-container>
+</template>
+
+<script>
+import transaction from '@/utils/transaction'
+import base58 from '@/libs/base58'
+import converters from '@/libs/converters'
+import TokenInfoModal from './TokenInfoModal'
+import SendToken from './SendToken'
+import IssueToken from './IssueToken'
+import BigNumber from 'bignumber.js'
+import BurnToken from './BurnToken'
+import { NODE_IP, SEND_FUNCIDX, SEND_FUNCIDX_SPLIT } from '../../../constants.js'
+import { CONTRACT_DESCRIPTOR, CONTRACT_WITH_SPLIT_DESCRIPTOR } from '../../../contract'
+import Vue from 'vue'
+import browser from '../../../utils/browser'
+export default {
+    name: 'TokenRecord',
+    components: { TokenInfoModal, SendToken, IssueToken, BurnToken },
+    data: function() {
```

Comment:
what is the usage of data in this file

Reply:
Use for showing token info in list and detail modal.